src/net/sourceforge/kolmafia/swingui/panel/GearChangePanel.java

```diff
@@ -0,0 +1,1497 @@
+package net.sourceforge.kolmafia.swingui.panel;
+
+import java.awt.*;
```

Comment:
we expand the *s on src/ files (or we try to, at least)

Reply:
thanks, this was my naughty IDE.